src/app/charity/charity.component.html

```diff
@@ -10,10 +10,30 @@ <h1 *ngIf="campaigns[0]" class="b-rh-1 b-bold">{{ campaigns[0].charity.name }} C
       </p>
     </div>
 
-    <div class="c-grid">
-      <div *ngFor="let campaign of campaigns">
-        <app-campaign-card [campaign]="campaign"></app-campaign-card>
-      </div>
-    </div>
+    <biggive-grid
+        slot="campaign-grid"
+        column-count="3"
+        *ngIf="campaigns"
+      >
+        <div *ngFor="let campaign of campaigns">
+          <biggive-campaign-card
+            id="campaign-{{campaign.id}}"
+            [banner]="campaign.imageUri | optimisedImage:830 | async"
+            [campaignType]="campaign.isMatched ? 'Match Funded' : null"
+            [campaignTitle]="campaign.title"
+            [donateButtonUrl]="'/donate/' + campaign.id"
+            [moreInfoButtonUrl]="'/campaign/' + campaign.id"
+            [organisationName]="campaign.charity.name"
+            primary-figure-label="Match Funds Remaining"
+            [primaryFigureAmount]="campaign.matchFundsRemaining | currency: campaign.currencyCode : 'symbol' : '1.0-0'"
+            [progressBarCounter]="getPercentageRaised(campaign)"
+            secondary-figure-label="Total Funds Raised"
+            [secondaryFigureAmount]="campaign.amountRaised | currency: campaign.currencyCode : 'symbol' : '1.0-0'"
```

Comment:
I'm not sure what '1.0-0' is. Maybe worth adding a comment to explain and/oror extracting it to give it a name.

Reply:
Done in https://github.com/thebiggive/donate-frontend/pull/1359/commits/20bb1dee6f9b55f6fb128857c6df46959e2a1dd6 and made everything in currency stats contexts use the new var